vendor/volcano.sh/apis/pkg/apis/scheduling/v1beta1/labels.go

```diff
@@ -46,6 +46,10 @@ const PodPreemptable = "volcano.sh/preemptable"
 // Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".
 const CooldownTime = "volcano.sh/cooldown-time"
 
+// MaxCooldownTimes is the annotation key of Pod to set maximum number of cooldown(evictions),
```

Comment:
Do not modify the files under vendor directly.

Reply:
that's my fault, let us discourse and decide to add this label in apis https://github.com/volcano-sh/apis/pull/84 repo.